python/lsst/pipe/tasks/extended_psf.py

```diff
@@ -28,22 +28,62 @@
     "StackBrightStarsTask",
     "MeasureExtendedPsfConfig",
     "MeasureExtendedPsfTask",
+    "DetectorsInRegion",
 ]
 
 from dataclasses import dataclass
-from typing import List
 
 from lsst.afw.fits import Fits, readMetadata
 from lsst.afw.image import ImageF, MaskedImageF, MaskX
 from lsst.afw.math import StatisticsControl, statisticsStack, stringToStatisticsProperty
 from lsst.daf.base import PropertyList
 from lsst.geom import Extent2I
-from lsst.pex.config import ChoiceField, Config, ConfigurableField, DictField, Field, ListField
+from lsst.pex.config import ChoiceField, Config, ConfigurableField, Field, ListField, ConfigDictField
 from lsst.pipe.base import PipelineTaskConfig, PipelineTaskConnections, Struct, Task
 from lsst.pipe.tasks.coaddBase import subBBoxIter
 from lsst.pipe.base.connectionTypes import Input, Output
 
 
+def find_region_for_detector(det_id, detectors_focal_plane_regions):
```

Comment:
It's also called inside `MeasureExtendedPsfTask`. I think we'd imagined this may be a somewhat convenient utility script allowing for a trivial lookup between detector ID and region ID. Perhaps that's not required?

Reply:
Oh, I see it now. I guess it's fine as it is.